foreman/foreman.spec

```diff
@@ -848,6 +852,7 @@ rm -rf %{buildroot}
 %{_datadir}/%{name}/bundler.d/jsonp.rb
 %exclude %{_datadir}/%{name}/bundler.d/openid.rb
 %exclude %{_datadir}/%{name}/bundler.d/test.rb
+%exclude %{_datadir}/%{name}/bundler.d/foreman_tasks.rb
```

Comment:
For the rpm packaging, the source of dependency specification is in the spec itself, so having more precision in the bundler file doesn't seem to be too much a benefit, I went for skpping it here and relying on the foreman-tasks one, as otherwise, the foreman-tasks plugin would not get it's depednency there for older foreman versions. On the other hand, as I mentioned above, the tasks is barely used on it's own, without other plugins or core, so I guess we could go both ways if needed.